controllers/ingressController.go

```diff
@@ -137,17 +121,18 @@ func WhiteListIP(c *gin.Context) {
 
 		lease := models.Leases{
 			UserID:    User.(*models.Users).ID,
-			LeaseIP:   ip,
+			LeaseIP:   enableUserResponse.Identifier,
 			LeaseType: "Ingress",
```

Comment:
shouldn't we change the lease type from `ingress` to `looker` for looker driver?

Reply:
done.